test-fuzz/tests/integration/ci.rs

```diff
@@ -28,6 +29,27 @@ fn dylint() {
         .success();
 }
 
+#[test]
+fn cargo_sort() {
+    Command::new("cargo")
+        .args(["install", "cargo-sort"])
+        .logged_assert()
+        .success();
```

Comment:
Rather than install `cargo-sort` here, please do so between these two lines in ci.yml: https://github.com/trailofbits/test-fuzz/blob/0b979220ee7ef54b89221cbe08fb0113db0c3fc7/.github/workflows/ci.yml#L165-L166